diff --git a/modules/migrate_drupal/src/Hook/MigrateDrupalHooks.php b/modules/migrate_drupal/src/Hook/MigrateDrupalHooks.php
index c52ab7b3f352786bc8dbb990ca61345306b35b1b..ad0e6020e9d3a3b1255659aaed03b6e552d3e741 100644
--- a/modules/migrate_drupal/src/Hook/MigrateDrupalHooks.php
+++ b/modules/migrate_drupal/src/Hook/MigrateDrupalHooks.php
@@ -114,6 +114,12 @@ public function migrationPluginsAlter(array &$definitions): void {
         $source_connection = $source_plugin->getDatabase();
         $version = NodeMigrateType::getLegacyDrupalVersion($source_connection);
       }
+      catch (RequirementsException) {
+        // This code currently runs whenever the definitions are being loaded
+        // and if you have a Drupal 8+ source site then the requirements will
+        // not be met for the system_site migration since the 'variables'
+        // subsystem was replaced with config entities in Drupal 8.
+      }
       catch (\Exception $e) {
         \Drupal::messenger()->addError(t('Failed to connect to your database server. The server reports the following message: %error.<ul><li>Is the database server running?</li><li>Does the database exist, and have you entered the correct database name?</li><li>Have you entered the correct username and password?</li><li>Have you entered the correct database hostname?</li></ul>', ['%error' => $e->getMessage()]));
       }
